js/attendeeContact.js

```diff
@@ -163,9 +164,10 @@ function incrementBadge() {
 
 
 //Builds the data object
-function attendee(id,name, ticket, activeBadges, badgeName, regStatus, artShowHold, opsHold, nonTransferName)
+function attendee(accountId, attendeeId, name, ticket, activeBadges, badgeName, regStatus, artShowHold, opsHold, nonTransferName)

```

Comment:
You expect accountId to still be important somewhere? We are saving it but see to do nothing more than print it on a debug console.log...